### PR TITLE
[MM-29441] Fix errcheck issues in web_hub.go

### PR DESCRIPTION
**Pull request by YahyaHaq**
**Summary**
Fix errcheck issues in server/channels/app/platform/web_hub.go

Ticket Link
https://github.com/mattermost/mattermost/issues/29441

### CODE REVIEW

File: server/channels/app/platform/web_hub.go [hunk 1]
```diff
@@ -150,7 +150,10 @@ func (ps *PlatformService) GetHubForUserId(userID string) *Hub {
 	// https://mattermost.atlassian.net/browse/MM-26629.
 	var hash maphash.Hash
 	hash.SetSeed(ps.hashSeed)
-	hash.Write([]byte(userID))
+	_, err := hash.Write([]byte(userID))
+	if err != nil {
+		ps.logger.Error("Unable to write userID to hash", mlog.Err(err))
```
Comment (hanzei) [inline server/channels/app/platform/web_hub.go:155]: Let's also log the userID

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the PR :+1:

File: server/channels/app/platform/web_hub.go (hunk 1 above)
Comment (YahyaHaq) [reply server/channels/app/platform/web_hub.go:155]: Sounds good

Comment (YahyaHaq) [reply server/channels/app/platform/web_hub.go]: @hanzei done

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Review by agnivade — APPROVED** [review]
FYI, this method actually never returns an error. It is just there to satisfy the interface.

But any ways, it's good to keep things consistent.

**Comment by agnivade** [comment]
@mvitale1989 - Looks like a flaky e2e test.

**Comment by mvitale1989** [comment]
> @mvitale1989 - Looks like a flaky e2e test.

It indeed looks that way, i'll look into it. Thanks for raising!

**Pull request merged**